app/main.py

```diff
@@ -1,3 +1,55 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int) -> Distance:
+        return Distance(self.km + other.km) \
+            if isinstance(other, Distance) \
+            else Distance(self.km + other)
```

Comment:
It's better to use standard `if` instead of ternary operator if it doesn't fit into 1 line

Fix everywhere